src/pages/api/stripe/billing/session.ts

```diff
@@ -30,6 +30,7 @@ const StripeCheckoutSession = async (
       if (subscription) {
         const price = get(first(subscription.items.data), 'price')
         const latestInvoice = get(subscription, 'latest_invoice')
+        const billingScheme = get(subscription, 'plan.billing_scheme')
```

Comment:
Extract billing scheme as a top-level value for ease of access in a couple places for conditional rendering.